css-view-transitions-2/Overview.bs

```diff
@@ -205,7 +205,22 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 		```
 	</div>
 
+# Pseudo-classes # {#pseudo-classes}
 
+## Active View Transition Pseudo-class '':active-view-transition()''' ## {#the-active-view-transition-pseudo}
+
+The <dfn id='active-view-transition-pseudo'>:active-view-transition(* | <<transition-types>>)</dfn> pseudo-class represents
+the root element of the document, if it has a matching [=active view transition=].
+It has the following syntax definition:
+
+<pre class=prod>
+	:active-view-transition(<<vt-type-selector>>)
+	<<vt-type-selector>> = '*' | <<custom-ident>>#
```

Comment:
This can be a comma separated list as per the resolution. This grammer doesn't capture that?

Reply:
The `#` means comma-separated